core/templates/dev/head/pages/exploration-editor-page/translation-tab/services/translation-language.service.spec.ts

```diff
@@ -53,11 +60,22 @@ describe('Translation language service', function() {
     });
 
     it('should not allow invalid state names to be set', function() {
-      tls.setActiveLanguageCode('eng');
+      expect(tls.setActiveLanguageCode('eng')).toBeNull();
```

Comment:
Why this change? Typically functions starting with "set" don't return anything. I don't think we should be testing that.

Reply:
You're right, I've fixed it! Done.